src/char_table.h

```diff
@@ -32,7 +35,9 @@ struct CharTable
       {
          return(0);
       }
-      if (ch < (int)ARRAY_SIZE(chars))
+      //if (ch < (int)ARRAY_SIZE(chars))
+      // Coverity: CID 76238 (#1 of 1): Out-of-bounds read (OVERRUN)
+      if (ch < CharTable_Length)
```

Comment:
I agree that this fix is wrong.

Coverity is likely complaining that ch is an int, which means it could be negative.
Maybe try 
```
if (ch > 0 && ch < ARRAY_SIZE(chars))
```